homeassistant/components/swiss_public_transport/__init__.py

```diff
@@ -76,11 +78,9 @@ async def async_migrate_entry(
         # This means the user has downgraded from a future version
```

Comment:
See https://github.com/home-assistant/core/blob/dev/tests/components/teslemetry/test_config_flow.py#L238 for example on testing downgrading